### ISSUE TITLE
color: #d3d3d3 --really?

### ISSUE THREAD

**Issue by rtm** [slikts/tooling#10]
Your font color selection is so out of compliance with ADA, or any contrast level readable by anyone with even moderately diminished ability to perceive low contrast letters, it's not funny.

At #d3d3d3, your contrast ratio is 1.5; it should be at least 3.0.

**Comment by slikts** [comment]
The [contrast ratio](https://contrast-ratio.com/#%23d3d3d3-on-%23283339) between `#d3d3d3` and `#283339` is 8.64, which conforms with WCAG AAA; a ratio of 1.5 would mean that the background would be white, which it isn't supposed to be. What environment are you using to get a white background?

**Comment by rtm** [comment]
My mistake, thanks for taking to the time to look at it and for your work on this site.
